Docs/source/index.rst

```diff
@@ -25,6 +25,7 @@ Welcome to the BeVolt BPS documentation!
    :caption: Software
 
    Software/Tasks
+   Software/RTOS
    Software/BSP
    Software/Drivers
    Software/Apps
```

Comment:
At the moment, index.rst isn't ordered properly. It would be nice if you could order the software layers from lowest to highest to it was easier to understand. (BSP -> Tasks)